extensions/Worker.Extensions.Kafka/src/KafkaOutputAttribute.cs

```diff
@@ -32,38 +32,38 @@ public KafkaOutputAttribute(string brokerList, string topic)
         /// Gets or sets the Avro schema.
         /// Should be used only if a generic record should be generated
         /// </summary>
-        public string? AvroSchema { get; set; }
+        public string AvroSchema { get; set; }
 
         /// <summary>
         /// Gets or sets the Maximum transmit message size. Default: 1MB
```

Comment:
nit: 
Use lowercase maximum. 
Consider using consistent casing across the file (ex: default or Default?).
Consider rephrasing to include the unit as well (like "in bytes").

Reply:
Done.